assets/src/components/form/select/model-select.js

```diff
@@ -21,6 +21,8 @@ import {
 	REACT_SELECT_TYPES,
 } from './default-select-configuration';
 
+export const MODEL_SELECT_LABEL_NONE = 'model-select-label-none';
```

Comment:
We talked about this being unnecessary.  It looks like it got added back in on this branch.

Reply:
No that was the "SELECT ALL" constants that added an element to the options array for choosing something other than the provided options.
This is for removing the `<label>` element that gets automatically added by the `ModelSelect` component and is incompatible with the WordPress admin layout which uses the `BaseControl` component to add the labels.

Reply:
I was specifically talking about the new `withBaseControl` HOC I created.  It removes the need to have this.